client/modules/post/components/UpdatePost.js.jsx

```diff
@@ -18,17 +20,18 @@ class UpdatePost extends Base {
     );
   }
   componentDidMount() {
-    console.log("Single item mounting");
+    // console.log("Single item mounting");
     // console.log(this.context);
 
     // action.fetchItem();
-    const populate = false;
+
     // const populate = false;
     const { dispatch, params } = this.props;
+    dispatch(userListActions.fetchList());
```

Comment:
I like having an example of the select in vanilla yote, but the user list api call is admin protected so this will be confusing to most users. We should remove it.